packages/api-clients/open-api/agreementApi.yml

```
@@ -910,6 +910,76 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Problem"
+  "/internal/agreements/{agreementId}":
```

Ok to not have the operation suffix for the delete because it's obvious (the method is DELETE), but for the archive, I would add `/archive` at the end because it's just a `POST` and we should be explicit about what it does...

Also, I propose to be more specific: these operations are very specific, they archive/delete after a delegation revocation...
Maybe we could be more specific in the routes and function names, like:

`DELETE /internal/delegationRevoked/agreements/{agreementId}`
`POST /internal/delegationRevoked/agreements/{agreementId}/archive`
```suggestion
  "/internal/agreements/{agreementId}/archive":
```